client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -165,66 +53,30 @@ class PublishingTools extends Component {
 		const { translate, siteIsJetpack, isAtomic } = this.props;
 
 		const renderPressThis = config.isEnabled( 'press-this' ) && ! this.isMobile();
-		const renderPostByEmail = siteIsJetpack;
 		const renderPostByVoice =
 			config.isEnabled( 'settings/post-by-voice' ) && ! siteIsJetpack && ! isAtomic;
 
-		if ( ! renderPressThis && ! renderPostByVoice && ! renderPostByEmail ) {
+		if ( ! renderPressThis && ! renderPostByVoice ) {
 			return;
 		}
 
 		return (
 			<div>
 				<SettingsSectionHeader title={ translate( 'Publishing Tools' ) } />
 
+				{ this.renderPostByEmailModule() }
 				{ renderPostByVoice && this.renderPostByVoiceModule() }
-				{ renderPostByEmail && this.renderPostByEmailModule() }
 				{ renderPressThis && this.renderPressThisModule() }
 			</div>
 		);
 	}
 }
 
-PublishingTools.defaultProps = {
-	isSavingSettings: false,
-	isRequestingSettings: true,
-	fields: {},
-};
-
 PublishingTools.propTypes = {
-	onSubmitForm: PropTypes.func.isRequired,
-	isSavingSettings: PropTypes.bool,
-	isRequestingSettings: PropTypes.bool,
 	fields: PropTypes.object,
+	isAtomic: PropTypes.bool,
+	siteId: PropTypes.number.isRequired,
+	siteIsJetpack: PropTypes.bool,
 };
 
-export default connect(
-	( state ) => {
-		const selectedSiteId = getSelectedSiteId( state );
-		const siteIsJetpack = isJetpackSite( state, selectedSiteId );
-		const isAtomic = isSiteAutomatedTransfer( state, selectedSiteId );
-		const regeneratingPostByEmail = isRegeneratingJetpackPostByEmail( state, selectedSiteId );
-		const siteInDevMode = isJetpackSiteInDevelopmentMode( state, selectedSiteId );
-		const moduleUnavailableInDevMode = isJetpackModuleUnavailableInDevelopmentMode(
-			state,
-			selectedSiteId,
-			'post-by-email'
-		);
-
-		return {
-			siteIsJetpack,
-			isAtomic,
-			selectedSiteId,
-			regeneratingPostByEmail,
-			postByEmailAddressModuleActive: !! isJetpackModuleActive(
-				state,
-				selectedSiteId,
-				'post-by-email'
-			),
-			moduleUnavailable: siteInDevMode && moduleUnavailableInDevMode,
-		};
-	},
-	{
-		regeneratePostByEmail,
-	}
-)( localize( PublishingTools ) );
```

Comment:
My goal here wasn't to remove the `connect` call. However, I discovered that several props were redundant (a lot of logic has been moved to `PostByEmailSetting`) and that `SiteSettingsFormWriting` can pass the other props we need.